requirements/README.md

```diff
@@ -0,0 +1,33 @@
+
+# Requirements Engineering
+The field of requirements engineering is a part of systems and software engineering, and deals with structuring requirements. Requirements are useful in measuring to what degree software solves the problems it was built to solve. Requirements engineering can be viewed in terms of two groups of activities, corresponding with the iterative and incremental nature of the process: requirements development and requirement management {{"pandey2010effective" | cite}}. To consider these activities, first a definition of a software requirement has to be established. Finally, a model for structuring the process of requirements engineering is considered.
+
+## The definition of a software requirement
+In the "Guide to the Software Engineering Body of Knowledge" (SWEBOK) the IEEE[^ieee] defines a software requirement as: 'a property that must be exhibited by something in order to solve some problem in the real world' {{"swebok14" | cite}}. Requirements can be imposed on the product as well as the process. An essential property of software requirements is that they are verifiable, either as an individual feature (functional requirement) or at the system level (non-functional requirement). In addition to this behavioral properties, requirements may have attributes such as a priority rating and a progression status.
+
+## Requirements development
+
+Discovering, analyzing, documenting and validating requirements are all part of the process of requirements development {{"pandey2010effective" | cite}}. This process is essential to defining a software architecture works as the end user expects. This part of the requirements engineering process takes place before a software implementation is developed. The goal is to end up with a specification of the requirements the software has to adhere to.
+
+One of the main challenges in requirements development in large systems is coordination {{"crowston1998coordination" | cite}}, specifically coordinating dependencies. Dependencies may arise between requirements, resources and software that consumes these resources. To manage this coordination mechanisms are needed, such as standardization, problem decomposition or (third-party) expert counsel. Early dependency detection and analysis is key to determining which coordination mechanism to apply.
+
+## Requirements management
+
+Throughout the software development process a requirement may change. This change may affect other parts of the system and in turn affect other requirements as well. Requirements management consists of activities related to managing these changes and tracing them to the corresponding parts of the system {{"pandey2010effective" | cite}}.
```

Comment:
I think may is fine here, since it's not a given that a requirement will change, but it could happen, and it even may happen, since there is a process for dealing with it. I don't mind changing it if you really think it should be changed though.